CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+Contributing code
+=================
+
+**Note: This document is just for getting started. Visit [**the wiki**](htt
+ps://github.com/cltk/cltk/wiki)
+for the full contributor's guide. Please be sure to read it carefully to make
+the code review process go as smoothly as possible and maximize the
+likelihood of your contribution being merged.**
+
+How to contribute
+-----------------
+
+The preferred way to contribute to cltk is to fork the 
+[main repository](http://github.com/cltk/cltk/) on
+GitHub. To get started, you will need a working installation
+of [Python 3.5](https://www.python.org/downloads/):
```

Comment:
We currently support 3.4 and 3.5 (though I would like to sunset 3.4 at some point in the next few months).

@nelson-liu I'll still get to merging this. I appreciate your patience. KJ